console/src/main/webapp/manager/app/assets/lang/fr.json

```diff
@@ -263,6 +263,7 @@
   "logs.roleremoved"      : "a été enlevé de l'utilisateur",
   "logs.emailrecoverysent": "Email de récupération du mot de passe envoyé",
   "logs.rolecreated"      : "Role créé",
+  "logs.oauth2usercreated": "Utilisateur OAuth2 créé",
```

Comment:
Thanks !
All lang files should be updated by the PR (using google translate if required)

Reply:
ok, will be done, thanks